requirements.txt

```diff
@@ -1,4 +1,4 @@
-pycodestyle==2.3.1
+pycodestyle~=2.4
```

Comment:
I'm not sure I understand how it was working before cause it seems version `1.4.3` of autopep did depend of this (https://github.com/hhatto/autopep8/commit/95832804e361588cb8e15d848a20089fbd8d9029#diff-2eeaed663bd0d25b7e608891384b7298)

Would it be better to stick to `autopep8` <= `1.4.3` ?

Reply:
Looks fine: I didn't see anything major going from 2.3 to 2.4.
It's impossible to be sure that no projects will tumble on that, but I think we should be going forward, not backward.
The minor version should be bumped.